Frontend/ui-driver/src/MerchantConfigs/Utils/Utils.purs

```diff
@@ -57,6 +57,11 @@ getMerchant unit = case (decodeMerchantId (getMerchantId "")) of
   Just merchant -> merchant
   Nothing -> NAMMAYATRIPARTNER
 
+getMerchantVehicleSize :: Unit -> Int
+getMerchantVehicleSize unit = 
+  case getMerchant unit of 
+    _ -> 90
+
```

Comment:
refactor this function

Reply:
@ashishsingh101 , this function is for future reference , like in future if driver vehicle size needs to be change , 
as it changed in Yatri app

Reply:
then you can pass constant value from purescript and later if required will create a function for this

Reply:
ok fine